rllib/agents/ars/ars.py

```diff
@@ -327,7 +327,7 @@ def cleanup(self):
 
     @override(Trainer)
     def compute_action(self, observation, *args, **kwargs):
-        action, _, _ = self.policy.compute_actions(observation, update=True)
+        action, _, _ = self.policy.compute_actions([observation], update=True)
```

Comment:
This behavior now matches the one of a "normal" Trainer's compute_action method: Pass a single obs in, then the method converts this into a 1-batch and calls the policy's compute_actions method.